code/go/0chain.net/smartcontract/storagesc/sc.go

```diff
@@ -125,16 +127,32 @@ func (ssc *StorageSmartContract) GetAddress() string {
 	return ADDRESS
 }
 
+func (ssc *StorageSmartContract) GetStat(name string) (int64, error) {
```

Comment:
you can't use metrics for it, the problem is that only MPT is protected by consensus, so we should store total number of authorizers in the different node with some hand crafted key for this value, similar to how we had ALL_BLOBBERS key before